common_rest/README.md

```diff
@@ -0,0 +1,26 @@
+# Error Handler Module
```

Comment:
Should we call it 'Common Rest Module' instead of 'Error Handler Module' here?

Reply:
Updated title and description.